doc/whats_new/v0.1.rst

```diff
@@ -27,6 +27,7 @@ Changelog
 ---------
 
 - |Feature| Implement partial correlation test :func:`pywhy_stats.fisherz`, by `Adam Li`_ (:pr:`7`)
+- |Feature| Add (un)conditional kernel independence test by `Patrick Blöbaum`_, co-authored by `Adam Li`_ (:pr:`14`)
```

Comment:
Can you dd your name to the authors list, so it links to your GH profile?

See: `what's_new/_contributors.rst`